README.md

```diff
@@ -406,6 +407,50 @@ Future.reject('Nothing broke').swap().fork(console.error, console.log);
 //> "Nothing broke"
 ```
 
+### Combining Futures
+
+#### and
+##### `#and :: Future a b ~> Future a b -> Future a b`
+##### `.and :: Future a b -> Future a b -> Future a b`
+
+Logical and for Futures.
+
+Returns a new Future which either rejects with the first rejection reason, or
+resolves with the last resolution value once and if both Futures resolve. This
+behaves analogues to how JavaScript's and operator does.
```

Comment:
This should either read:

  - _This behaves analogously to…_; or
  - _This behaviour is analogous to…_.